package-testing/sdk-test-runner/.env

```diff
@@ -0,0 +1,5 @@
+TEST_CASE_FILE="../scenarios.json"
+SDK_SERVER_HOST="http://localhost"
+SDK_SERVER_PORT=4000
+API_SERVER_HOST="http://localhost"
+API_SERVER_PORT=5000
```

Comment:
I think this is what we'll have to change to hit cloud function ([link](https://console.cloud.google.com/functions/details/us-central1/serveGitHubRacTestFile?env=gen2&project=eppo-qa&tab=source)) for android